core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -40,6 +41,21 @@ oppia.directive('simpleEditorSidebar', [
           $scope.questionList = SimpleEditorManagerService.getQuestionList();
           $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
           $scope.sidebarModeService = SimpleEditorSidebarModeService;
+
+          $scope.canAddNewQuestion = (
+            SimpleEditorManagerService.canAddNewQuestion);
+
+          $scope.addNewQuestion = function() {
+            if (!SimpleEditorManagerService.canAddNewQuestion()) {
+              return;
+            }
+//            $scope.sidebarModeService.setModeToReadonly();
```

Comment:
Is this supposed to be removed or left in? Let's please do either of those, but not comment out the code like this.